### ISSUE TITLE
The applet's icon doesn't respond to left/right click.

### ISSUE THREAD

**Issue by cazacugmihai** [serdarsen/budgie-advanced-brightness-controller#8]
**Issue**
```
The applet's icon doesn't respond to left/right click.
```

**Error**

```
budgie-wm.desktop[1651]: message repeated 8 times: [ Window manager warning: Overwriting existing binding of keysym ff61 with keysym ff61 (keycode 6b).]
budgie-panel.desktop[1659]: Error: Need to be called with exactly two arguments
budgie-panel.desktop[1659]: Usage: /usr/lib/gnome-settings-daemon/gsd-backlight-helper device brightness
budgie-panel.desktop[1659]:   device:      The backlight directory starting with "/sys/class/backlight/"
budgie-panel.desktop[1659]:   brightness:  The new brightness to write
```

**Envirnoment**
```
OS: Ubuntu 19.04 x86_64
DE: Budgie 
WM: Mutter(Budgie)
```

**Comment by phenonymous** [comment]
I've tried to find a solution to this since I was having the same issue. See #9 why it's happening. 
So `gsd-backlight-helper` is just a plugin for `gnome-settings-daemon`, which is what `budgie-desktop-settings` is using, and it was never intended to be used by other programs.
When you go to power settings and change brightness there, `gsd-backlight-helper` gets called and sets the brightness. They changed it so that `gsd-power` retrieves the brightness values internally which is why you get the above error.

Now I don't know the proper way to handle it - here's how I solved it;

List the devices in `/sys/class/backlight`. For each device listed there, check their `max_brightness`
`$ cat /sys/class/backlight/example/max_brightness`. Take the name of the device that has the lowest max brightness and replace the following lines in `/usr/lib/budgie-desktop/plugins/budgie-advanced-brightness-controller/BackLightHelper.py` - replace `acpi_video0` with your device.

```diff
diff --git a/src/budgie-advanced-brightness-controller/BackLightHelper.py b/src/budgie-advanced-brightness-controller/BackLightHelper.py
index 6420414..6a85045 100644
--- a/src/budgie-advanced-brightness-controller/BackLightHelper.py
+++ b/src/budgie-advanced-brightness-controller/BackLightHelper.py
@@ -39,7 +39,7 @@ class BackLightHelper():
         maxBrightness = 0
         try:
             p = subprocess.Popen(
-                ['pkexec', '/usr/lib/gnome-settings-daemon/gsd-backlight-helper', '--get-max-brightness'],
+                ['cat', '/sys/class/backlight/acpi_video0/max_brightness'],
                 stdout=subprocess.PIPE)
             comm = p.communicate()[0]
             maxBrightness = int(comm)
@@ -54,7 +54,9 @@ class BackLightHelper():
 
     def getCurrentBrightnessFromHelper(self):
         try:
-            p = subprocess.Popen(['pkexec', '/usr/lib/gnome-settings-daemon/gsd-backlight-helper', '--get-brightness'], stdout=subprocess.PIPE)
+            p = subprocess.Popen(
+                ['cat', '/sys/class/backlight/acpi_video0/brightness'],
+                stdout=subprocess.PIPE)
             comm = p.communicate()[0]
             self.currentBrightness = int(comm)
         except:
@@ -71,7 +73,7 @@ class BackLightHelper():
     def setBrightness(self, brightness):
         self.currentBrightness = brightness
         try:
-            subprocess.call(['pkexec', '/usr/lib/gnome-settings-daemon/gsd-backlight-helper', '--set-brightness', "%s" % self.currentBrightness])
+            subprocess.call(['pkexec', '/usr/lib/gnome-settings-daemon/gsd-backlight-helper', '/sys/class/backlight/acpi_video0', "%s" % self.currentBrightness])
         except:
             MyLog.e(self.TAG, "Error:6033")
             self.setAvailable(False)
```

**Comment by serdarsen** [comment]
Thank you @phenonymous your suggestions helped me a lot.  I fixed those in v1.3.0
